src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -84,7 +85,7 @@ public abstract class CheckinReceivePiecesHelper<T> extends BaseHelper {
   private final Map<String, Map<String, Error>> processingErrors;
   private final Set<String> processedHoldingsParams;
   private final Map<String, String> processedHoldings;
-
+  private static final String BARCODE_NOT_UNIQUE_MESSAGE = "Barcode must be unique";
```

Comment:
better to declare constants at the top of class before declaring other variables